api/specs/webserver/v0/components/schemas/project-progress-v0.0.1.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://simcore.io/api/specs/webserver/v0/components/schemas/project-progress-v0.0.1.json",
+  "title": "project progress/status",
+  "description": "Progress related to the status of each node inputs/outputs",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "nodes"
+  ],
+  "properties": {
+    "uuid": {
+      "type": "string",
+      "description": "project unique identifier",
+      "examples": [
+        "07640335-a91f-468c-ab69-a374fa82078d",
+        "9bcf8feb-c1b1-41b6-b201-639cd6ccdba8"
+      ]
+    },
+    "workbench": {
+      "type": "object",
+      "patternProperties": {
+        "^\\S+$": {
```

Comment:
Shouldn't this pattern be the same as the one below?

Reply:
no that one means anything but spaces. the other I think is related to uuids